app/src/main/java/com/github/palFinderTeam/palfinder/utils/time/Time.kt

```diff
@@ -105,4 +106,26 @@ fun askDate(supportFragmentManager: FragmentManager, date: SimpleDate? = null):
     return dateFrag.value.thenApply {
         it.withTime(SimpleTime(0,0))
     }
-}
\ No newline at end of file
+}
+
+/**
+ * Get the maximum starting date for a meetup
+ */
+val maxStartDate: Calendar
+    get() {
+        val maxDate = Calendar.getInstance()
+        //event start max 5 days after today
+        maxDate.add(Calendar.DAY_OF_MONTH, 7)
```

Comment:
Could name this constant somewhere and the comment does not match, but otherwise it is great.

Reply:
+1